domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -570,6 +573,7 @@ class ExplorationProgressController @Inject constructor(
         recomputeCurrentStateAndNotifyAsync()
       }.launchIn(CoroutineScope(backgroundCoroutineDispatcher))
       explorationProgress.advancePlayStageTo(LOADING_EXPLORATION)
+      explorationProgressListeners.forEach(ExplorationProgressListener::onExplorationSessionStarted)
```

Comment:
For this & the below callback: does exploration ID never need to be passed?

Reply:
At the moment, I am not using the exploration ID for any logic, which is why I am not passing it.
The granularity currently required is at topic id level.